api_tests/waffle/views/test_waffle_list.py

```diff
@@ -74,3 +78,28 @@ def test_waffle_switch_logged_out(self, app, user, url, inactive_switch):
     def test_page_size(self, app, url, user):
         res = app.get(url)
         assert res.json['links']['meta']['per_page'] == MaxSizePagination.page_size
+
+
+@pytest.mark.django_db
+class TestWaffleCookies(OsfTestCase):
+
+    @pytest.mark.enable_bookmark_creation
+    def test_waffle_leaves_cookie(self):
```

Comment:
I could not think of any expedient way of testing this without relying on a real page where this happens.